### ISSUE TITLE
View Due Jobs only shows Job Status and Due Date

### ISSUE THREAD

**Issue by BrandonFirmstone** [BrandonFirmstone/ProjectTerminal#9]
**Describe the bug**
When trying to view due jobs only the job status and due date is shown to the user. This makes the function essentially useless because the user needs to know the descriptions and priorities in order to make informed decisions.

**Expected behavior**
The expected behaviour is that it should show all of the information to the user.

**Screenshots**

![image](https://user-images.githubusercontent.com/83018530/133714628-47bea765-28f1-487a-91c1-f5a5bd981616.png)

**Comment by BrandonFirmstone** [comment]
Attempt 1:
My first thought is that perhaps it's due to the way that it's reading the jobs.csv file. So, to test this I changed the all_jobs variable to read directly from the CSV. This also made me notice that it's only doing this in the terminal displayed using Heroku, not Gitpod - the IDE I'm using. I changed the variable neverless to instead call the function get_all_jobs() to read directly from the csv file. all_jobs = pd.read_csv("jobs.csv")
